basics/README.adoc

```diff
@@ -28,7 +28,6 @@ The following are the foundational requirements for uProtocol specifications:
 * *link:qos.adoc[Quality of Service & Priority (UPriority)]*
 * *link:uattributes.adoc[uProtocol Attributes (UAttributes)]*
 * *link:umessage.adoc[uProtocol Message (UMessage)]*
-* *link:upayload.adoc[uProtocol Payload (UPayload)]*
```

Comment:
maybe replace this with the link to upayloadformat.adoc?

Reply:
done